app.js

```diff
@@ -99,16 +103,19 @@ function generateImg() {
   var leftImgElement = document.createElement('img');
   leftImgElement.setAttribute('src', allProducts[previousImgs[0]].path);
   leftImgElement.setAttribute('alt',allProducts[previousImgs[0]].name);
+  allProducts[previousImgs[0]].views++;
   imgContainer.appendChild(leftImgElement);
 
   var centerImgElement = document.createElement('img');
   centerImgElement.setAttribute('src', allProducts[previousImgs[1]].path);
   centerImgElement.setAttribute('id', allProducts[previousImgs[1]].name);
+  allProducts[previousImgs[1]].views++;
```

Comment:
If the only differences between these three blocks is the index, can you think of a way to do them with a loop?